src/references/ilkList.js

```diff
@@ -44,11 +44,11 @@ export default [
   //   gem: 'COL4',
   //   currency: COL4
   // },
-  // {
-  //   slug: 'col5-a',
-  //   symbol: 'COL5-A',
-  //   key: 'COL5-A',
-  //   gem: 'COL5',
-  //   currency: COL5
-  // }
+  {
+    slug: 'col5-a',
+    symbol: 'COL5-A',
+    key: 'COL5-A',
+    gem: 'COL5',
+    currency: COL5
+  }
```

Comment:
Are we not adding COL2-4? 🤔

Reply:
They weren't in the testnet.json, which was causing problems with the mcd plugin, but with @ethanbennett's PR here, they will be ready to be added in https://github.com/makerdao/dai.js/pull/127/files#diff-9d66bae500be515575fa289f1d9ebcd4R4

Reply:
@abylaw I just published a new version of the MCD plugin which allows us to use the rest of these COLs, so COL1-5 are available to use now :-)